src/handler.ts

```diff
@@ -104,17 +105,31 @@ abstract class AbstractFileHandler implements IMessageFilter {
   }
 
   /**
-   * Open a path and return a widget.
+   * A signal emitted when the active widget changes.
+   */
+  get activeChanged(): ISignal<AbstractFileHandler, IChangedArgs<Widget>> {
```

Comment:
actually, this should just be an `activated` signal. The document manager doesn't care when it deactivates, only when it activates.